miden-lib/src/transaction/inputs.rs

```diff
@@ -21,9 +23,13 @@ pub(super) fn extend_advice_inputs(
     tx_args: &TransactionArgs,
     advice_inputs: &mut AdviceInputs,
 ) {
-    build_advice_stack(tx_inputs, tx_args.tx_script(), advice_inputs);
+    // TODO: remove this value and use a user input instead
+    let kernel_version = 0;
```

Comment:
Let's create an issue for this. I think we should define kernel version when we instantiate the transaction executor.

Reply:
Created an issue: #874